specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/managedHsm.json

```diff
@@ -392,7 +392,7 @@
               "$ref": "#/definitions/MHSMPrivateEndpointConnection"
             }
           },
-          "204": {
+          "404": {
```

Comment:
>404 [](start = 11, length = 3)

Removing the 204 is correct and your service should be returning 404 to the GET if the resource doesn't exist. However all errors are taken care of by the default response. If you explicitly declare the 404 response, it will change how your SDK is generated and in most cases makes the SDK inconsistent with other SDKs. Unless you have a specific reason to declare the 404 response, I'd leave it out.

Reply:
I don’t have specific reason for 404 response, removed that.